packages/web/src/common/store/cache/collections/utils/retrieveCollections.ts

```diff
@@ -86,7 +86,7 @@ export function* retrieveTracksForCollections(
  * Retrieves a single collection via API client
  * @param playlistId
  */
-function* retrieveCollection(playlistId: ID) {
+export function* retrieveCollection(playlistId: ID) {
```

Comment:
wow surprised this wasn't exported before

Reply:
yeah I think until now we've always use the `retrieveCollections` function (for which this one is a helper) below which tries to get the playlist from cache first/automatically caches the result after